<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="8,0">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="5,1">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="Video" name="videoOutPut" native="true">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1">
              <item>
               <widget class="QPushButton" name="pushButton_3">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>pause</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="slider_process">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_2">
                <property name="minimumSize">
                 <size>
                  <width>71</width>
                  <height>41</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>speed</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Volumn</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="slider_Volumn">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>brightness</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="horizontalSlider_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="pushButton_4">
                <property name="minimumSize">
                 <size>
                  <width>71</width>
                  <height>41</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>71</width>
                  <height>41</height>
                 </size>
                </property>
                <property name="text">
                 <string>open</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="minimumSize">
                 <size>
                  <width>71</width>
                  <height>41</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>71</width>
                  <height>41</height>
                 </size>
                </property>
                <property name="text">
                 <string>fullDisplay</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>78</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>278</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Video</class>
   <extends>QWidget</extends>
   <header location="global">video.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
